package com.pratian.appointmentservice.exceptions;

public class TestNotFoundException extends RuntimeException {

	public TestNotFoundException() {
		super();
		// TODO Auto-generated constructor stub
	}

	public TestNotFoundException(String message, Throwable cause, boolean enableSuppression,
			boolean writableStackTrace) {
		super(message, cause, enableSuppression, writableStackTrace);
		// TODO Auto-generated constructor stub
	}

	public TestNotFoundException(String message, Throwable cause) {
		super(message, cause);
		// TODO Auto-generated constructor stub
	}

	public TestNotFoundException(String message) {
		super(message);
		// TODO Auto-generated constructor stub
	}

	public TestNotFoundException(Throwable cause) {
		super(cause);
		// TODO Auto-generated constructor stub
	}
	
	

}
